SteamKit2/SteamKit2/Steam/Discovery/IServerListProvider.cs

```diff
@@ -8,6 +9,14 @@ namespace SteamKit2.Discovery
     /// </summary>
     public interface IServerListProvider
     {
+        /// <summary>
+        /// When the server list was last refreshed, used to determine if the server list should be refreshed from the Steam Directory
+        /// </summary>
+        /// <remarks>
+        /// This should return DateTime with the UTC kind
+        /// </remarks>
+        DateTime LastServerListRefresh { get; }
```

Comment:
Breaking change, can't we make use of interfaces default implementation and make it `=> DateTime.MinValue` or something by default? I like how we force it from the customers, that's the way it should be done tbh, but we have alternative if needed.

Reply:
Returning MinValue would change the behaviour (it would always try to refresh before using the provided list)